inc/draw.h

```diff
@@ -4,4 +4,10 @@
  * */
 #include "utilities.h"
 
- E_OpResult Draw_ModuleInit(void);
\ No newline at end of file
+ E_OpResult Draw_ModuleInit(void);
+
+
+int decrementPlayerXCoord(void);
```

Comment:
In fact, that is not incrementation/decrementation. Further development should change the name of this function because it does not do what is percieved from it's name

Reply:
Done